packages/client/package.json

```diff
@@ -42,9 +42,9 @@
     "prepublishOnly": "../../config/cli/prepublish.sh",
     "repl": "npx tsx ./bin/repl.ts --logLevel=error",
     "test": "npm run test:unit && npm run test:integration",
-    "test:cli": "npx vitest run ./test/cli/*.spec.ts",
-    "test:integration": "npx vitest run ./test/integration/*.spec.ts",
-    "test:unit": "npx vitest run test/* -c=./vitest.config.unit.ts",
+    "test:cli": "npx vitest run -c ../../config/vitest.config.mts ./test/cli/*.spec.ts",
+    "test:integration": "npx vitest run -c ../../config/vitest.config.mts ./test/integration/*.spec.ts",
+    "test:unit": "npx vitest run -c ../../config/vitest.config.mts test/*",
```

Comment:
Ah! This one was `./vitest.config.unit.ts` but now points to the monorepo config "base vitest config". Can you do the same here as with the VM package, expanding the package-level config with the monorepo-level config?